src/documents/management/commands/manage_superuser.py

```diff
@@ -1,42 +1,64 @@
 import logging
 import os
-
+import re
 from django.contrib.auth.models import User
 from django.core.management.base import BaseCommand, CommandError
-
+from django.forms import EmailField
```

Comment:
Why not use the validator directly? No need to import django.forms here ;)
```suggestion
from django.core.validators import EmailValidator
```